### DISCUSSION TITLE
Quick question - what's the knob controller in the product photos?

### DISCUSSION THREAD

**Discussion by weavermedia** [shorepine/tulipcc#284] [General]
What is the 8 knob controller in this image? Looks cool!

![tulipcc-waves](https://github.com/user-attachments/assets/09683837-0e94-4bf4-beaa-5e9b7c5db5e5)

**Comment by bwhitman** [comment]
That's a m5stack 8angle! 

https://shop.m5stack.com/products/8-angle-unit-with-potentiometer

See more supported devices here 

https://github.com/shorepine/tulipcc/blob/main/docs/tulip_api.md#i2c--grove--mabee
